serving/samples/rest-api-go/README.md

```diff
@@ -1,61 +1,76 @@
 # Creating a RESTful Service
 
-This sample demonstrates creating a simple RESTful service. The exposed endpoint
-takes a stock ticker (i.e. stock symbol), then outputs the stock price. The
-endpoint resource name is defined by an environment variable set in the
-configuration file.
+This sample demonstrates creating and running a simple RESTful service on Knative Serving.
+The exposed endpoint takes a stock ticker (i.e. stock symbol), then outputs the stock price.
 
 ## Prerequisites
 
 1. A Kubernetes cluster with
-   [Knative Serving](https://github.com/knative/docs/blob/master/install/README.md)
-   installed.
-2. Install
-   [Docker](https://docs.docker.com/get-started/#prepare-your-docker-environment).
-3. You need to
-   [configure outbound network access](https://github.com/knative/docs/blob/master/serving/outbound-network-access.md)
-   because this application makes an external API request.
-4. Check out the code:
+   [Knative Serving](https://github.com/knative/docs/blob/master/install/README.md) v0.3 or higher installed.
+2. [Docker](https://docs.docker.com/get-started/#prepare-your-docker-environment) installed locally.
```

Comment:
All numbered lists can use just "1." for each point, and it will be automatically numbered correctly when the file is rendered. This is very useful when adding/deleting steps later on; you'll never have to renumber a list.

Reply:
Thanks for the review. Fixed the numbering.